server/models/index.js

```diff
@@ -0,0 +1,39 @@
+
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/../config/config.json`)[env];
```

Comment:
I tried importing this config variable using es6 syntax. it keeps giving an error. Any workabout?